### DISCUSSION TITLE
IconButton Tooltips

### DISCUSSION THREAD

**Discussion by S-Bra** [iTwin/iTwinUI#2182] [General]
Saw the recommendation on IconButton label as the preferred way of providing a tooltip.
This sort of conflicts with the standard mechanism to provide the tooltip via the title property. At the moment you can use both and have the freedom to choose between them. 

![image](https://github.com/user-attachments/assets/88d15c85-d01c-4d2d-a882-6b2694f642f8)

https://stackblitz.com/edit/github-21dait-zjtggy?file=src%2FApp.tsx

Suggestion to avoid a duplicate tooltip:
Either address at build time with typings that prevent having a tile property while the label property is present.
Or address at runtime and ignore the title property if the label is present.

**Comment by mayank99** [comment]
This has been addressed in [3.14.0](https://github.com/iTwin/iTwinUI/releases/tag/%40itwin%2Fitwinui-react%403.14.0) 🚀

- `title` prop has been deprecated and turned into an alias for `label`
- a warning will be displayed when `label` (or `title`) is not used

**Comment by S-Bra** [comment]
[like]  Stephan Braeuer reacted to your message:
________________________________
